### PR TITLE
Divine Intervention

### PR DESCRIPTION
**Pull request by mjmagee991**
Modified the code to work with the salvaged TMS

### CODE REVIEW

File: include/TMS/TMS.hpp [hunk 1]
```diff
@@ -49,7 +51,10 @@ class TMS {
     /**
      * Array to store the thermistor objects
      */
-    DEV::Thermistor thermistors[NUM_THERMISTORS];
+    DEV::Thermistor thermistor;
+
+    IO::GPIO* mux1;
```
Comment (cbolles) [inline include/TMS/TMS.hpp]: Can we make these references?

**Review by cbolles — CHANGES_REQUESTED** [review]
Looks good, just a couple of places we can use references instead of pointers

**Comment by AarC10** [comment]
Seems legit

**Review by AarC10 — APPROVED** [review]
Seems legit Part 2: The Review

**Review by cbolles — APPROVED** [review]

**Pull request merged**